Sources/SpeziScheduler/Notifications/SchedulerNotifications.swift

```diff
@@ -24,6 +24,48 @@ import struct SwiftUI.AppStorage
 ///
 /// Notifications can be automatically scheduled for Tasks that are scheduled using the ``Scheduler`` module. You configure a Task for automatic notification scheduling by
 /// setting the ``Task/scheduleNotifications`` property.
+/// 
+/// When notifications are enabled, the app will send alerts based on the task’s schedule. 
+///     If you want to stop notifications for a specific event, you can call ``Event/complete()`` to mark the event as complete.
+///     This will prevent notifications only for that particular occurrence, while other events will still trigger notifications as scheduled.
+///
+/// Below is an example of how to schedule a task with notifications and selectively disable notifications for certain events:
+///
+/// ```swift
+/// // Step 1: Schedule a recurring task with notifications
+/// try scheduler.createOrUpdateTask(
+///     id: "task-id",
+///     title: "Daily Task Reminder",
+///     instructions: "Remember to complete your daily task!",
+///     category: .questionnaire,
+///     schedule: .daily(hour: 9, minute: 0, startingAt: .today),
+///     scheduleNotifications: true
+/// )
+///
+/// // Step 2: Mark upcoming events as complete to stop notifications
+/// // Suppose you want to disable notifications for the next `daysAhead` days, you can complete those events in advance.
+/// @MainActor
+/// func disableNotificationForEvent(for taskID: String, daysAhead: Int) {
+///     let today = Date()
+///     guard let endDate = Calendar.current.date(byAdding: .day, value: daysAhead, to: today) else { return }
+///
+///     do {
+///         let events = try scheduler.queryEvents(for: today..<endDate, predicate: #Predicate { $0.id == taskID })
+///         for event in events {
+///             try event.complete()
+///             print("Marked event on \(event.occurrence.start) as complete. Notification disabled for this event.")
+///         }
+///     } catch {
+///         print("Error completing events: \(error)")
+///     }
+/// }
```

Comment:
We might even make this a bit more compact and ensure that the surrounding code will have to do the error handling:
```suggestion
/// @MainActor
/// func disableNotificationForEvent(for taskID: String, daysAhead: Int) throws {
///     guard let endDate = Calendar.current.date(byAdding: .day, value: daysAhead, to: .now) else { return }
///
///     let events = try scheduler.queryEvents(for: today..<endDate, predicate: #Predicate { $0.id == taskID })
///     for event in events {
///         try event.complete()
///     }
/// }
```